internal/store/node_test.go

```diff
@@ -68,7 +68,7 @@ func TestNodeStore(t *testing.T) {
 					Name:              "127.0.0.1",
 					CreationTimestamp: metav1.Time{Time: time.Unix(1500000000, 0)},
 					Labels: map[string]string{
-						"type": "master",
+						"node-role.kubernetes.io/master": "true",
```

Comment:
This is never true, I think this is always empty.

Reply:
I've seen in both ways (e.g. in OpenShift the convention is to use "true"), but empty is a better test :)